apps/explorer/src/hooks/useGetTransactionBlocks.ts

```diff
@@ -13,6 +13,7 @@ export function useGetTransactionBlocks(
 	filter?: TransactionFilter,
 	limit = DEFAULT_TRANSACTIONS_LIMIT,
 	refetchInterval?: number,
+	disabled?: boolean,
```

Comment:
This is confusing, can you move this to be an object, and just mirror the `useQuery` hook param. Or use `useSuiClientQuery`.